### DISCUSSION TITLE
[Stem] Expanding the scope of Vike Extensions

### DISCUSSION THREAD

**Discussion by brillout** [magne4000/universal-middleware#8] [General]
With @nitedani we were thinking of having zero-config integrations for servers (`vike-express`, `vike-hono`, ...).

I think we can (and should!) also integrate Universal Middlewares into these server integrations.

```js
import express from 'express'
import vike from 'vike-express'
const app = express()

// Integrates not only Vike's renderPage() but also the server endpoint of
// Telefunc/tRPC/GraphQL/...
app.use(vike)
```

In other words:
- Vike extensions are full-stack integrations (not only frontend)
- Vike extensions *are* Stem. There won't be any npm package nor GitHub repositories called "Stem".

Vike Core => only Frontend  
Vike Extensions => full-stack zero-config ejectable integrations

**Comment by magne4000** [comment]
WIP in https://github.com/vikejs/vike-node/issues/62
